lib/cam.js

```diff
@@ -196,15 +198,42 @@ Cam.prototype.connect = function(callback) {
  * @param {object} options
  * @param {string} [options.service] Name of service (ptz, media, etc)
  * @param {string} options.body SOAP body
- * @param {string} [options.url] Defines another url to request
- * @param {boolean} [options.ptz] make request to PTZ uri or not
+ * @param {string} [options.url] Defines another url to request instead of using the URLs from GetCapabilities/GetServices
  * @param {Cam~RequestCallback} callback response callback
  * @private
  */
 Cam.prototype._request = function(options, callback) {
+	const _this = this;
```

Comment:
I think that if we already use reverse apostrophe (`) for the strings, we can use here arrow callback functions. The same thing was when in some parts of the code we started to use `let` instead of `var`. There is no need to support old node.js versions, I suppose. Just we need to check and update the version of the node engine in the package.json.